src/js/src/store/modules/ngram.store.js

```diff
@@ -0,0 +1,116 @@
+import { requestService } from '../../services/RequestService'
+
+
+// Global ngram state.
+const initialState = () => ({
+  query: '',
+  // cache for current query in case server rejects it
+  attemptedQuery:'',
```

Comment:
You don't use this attemptedQuery anywhere? At least I can't seem to find it...

Reply:
Yes, this is dead code - good catch 👍